src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/shoppinglists/ShoppingListSyncIT.java

```diff
@@ -89,167 +103,62 @@ void sync_WithoutUpdates_ShouldReturnProperStatistics() {
             .getReportMessage())
-            .isEqualTo("Summary: 1 shopping lists were processed in total "
-                    +  "(1 created, 0 updated and 0 failed to sync).");
+            .isEqualTo("Summary: w shopping lists were processed in total "
+                + "(1 created, 0 updated and 0 failed to sync).");
     }
 
     @Test
-    void sync_WithUpdatedShoppingList_ShouldReturnProperStatistics() {
-
-        createShoppingList(CTP_SOURCE_CLIENT, "name-2", "key-2", "desc-2",
-                "anonymousId-2", "slug-2", 180);
-        createShoppingList(CTP_TARGET_CLIENT, "name-2", "key-2");
-
+    void sync_WithUpdatedCustomerOnShoppingList_ShouldReturnProperStatistics() {
         final List<ShoppingList> shoppingLists = CTP_SOURCE_CLIENT
-                .execute(buildShoppingListQuery())
-                .toCompletableFuture()
-                .join()
-                .getResults();
-
-        final List<ShoppingListDraft> shoppingListDrafts = mapToShoppingListDrafts(shoppingLists);
-
-        final ShoppingListSyncStatistics shoppingListSyncStatistics = shoppingListSync
-                .sync(shoppingListDrafts)
-                .toCompletableFuture()
-                .join();
-
-        assertThat(errorMessages).isEmpty();
-        assertThat(exceptions).isEmpty();
-
-        AssertionsForStatistics.assertThat(shoppingListSyncStatistics).hasValues(2, 1, 1, 0);
-        assertThat(shoppingListSyncStatistics
-                .getReportMessage())
-                .isEqualTo("Summary: 2 shopping lists were processed in total "
-                        + "(1 created, 1 updated and 0 failed to sync).");
-    }
-
-    @Test
-    void sync_WithUpdatedCustomerReference_ShouldReturnProperStatistics() {
-
-        final CustomerDraft customerDraft =
-                CustomerDraftBuilder.of("dummy-email", "dummy-password")
-                                    .key("dummy-customer-key")
-                                    .build();
-
-        final Customer sourceCustomer =
-                createCustomer(CTP_SOURCE_CLIENT, customerDraft);
-
-        createCustomer(CTP_TARGET_CLIENT, customerDraft);
-        createShoppingListWithCustomer(CTP_SOURCE_CLIENT, "name-2", "key-2", sourceCustomer);
-        createShoppingList(CTP_TARGET_CLIENT, "name-2", "key-2");
+            .execute(buildShoppingListQuery())
+            .toCompletableFuture()
+            .join()
+            .getResults();
 
-        final List<ShoppingList> shoppingLists = CTP_SOURCE_CLIENT
-                .execute(buildShoppingListQuery())
-                .toCompletableFuture()
-                .join()
-                .getResults();
+        createSampleCustomerJaneDoe(CTP_SOURCE_CLIENT);
+        final Customer sampleCustomerJaneDoe = createSampleCustomerJaneDoe(CTP_TARGET_CLIENT);
 
-        final List<ShoppingListDraft> shoppingListDrafts = mapToShoppingListDrafts(shoppingLists);
+        final List<ShoppingListDraft> updatedShoppingListDrafts =
```

Comment:
Do you also want to check the line items reordering here or just with one test?

Reply:
Would be good, but we have enough coverage, so imo not necessary.